.github/CODEOWNERS

```diff
@@ -16,28 +16,29 @@
 # docs/  docs@example.com
 
 #### Build stuff
-scripts/ @microsoft/fluentui-react-build
-package.json @microsoft/fluentui-react-build
-yarn.lock @microsoft/fluentui-react-build
-*.config.js @microsoft/fluentui-react-build
-*.sh @microsoft/fluentui-react-build
-*.yml @microsoft/fluentui-react-build
-tsconfig.json @microsoft/fluentui-react-build
-lerna.json @microsoft/fluentui-react-build
-.codesandbox @microsoft/fluentui-react-build
-.devops @microsoft/fluentui-react-build
-.eslintrc.* @microsoft/fluentui-react-build
-.gitattributes @microsoft/fluentui-react-build
-.githooks @microsoft/fluentui-react-build
-.gitignore @microsoft/fluentui-react-build
-.npmrc @microsoft/fluentui-react-build
-.npmignore @microsoft/fluentui-react-build
-.vscode @microsoft/fluentui-react-build
-sizeauditor.json @microsoft/fluentui-react-build
+/scripts/ @microsoft/fluentui-react-build
+/package.json @microsoft/fluentui-react-build
+/yarn.lock @microsoft/fluentui-react-build
+/*.config.js @microsoft/fluentui-react-build
+/*.sh @microsoft/fluentui-react-build
+/*.yml @microsoft/fluentui-react-build
+/tsconfig.json @microsoft/fluentui-react-build
+/lerna.json @microsoft/fluentui-react-build
+/.codesandbox @microsoft/fluentui-react-build
+/.devops @microsoft/fluentui-react-build
+/.eslintrc.* @microsoft/fluentui-react-build
+/.gitattributes @microsoft/fluentui-react-build
+/.githooks @microsoft/fluentui-react-build
+/.gitignore @microsoft/fluentui-react-build
+/.npmrc @microsoft/fluentui-react-build
+/.npmignore @microsoft/fluentui-react-build
+/.vscode @microsoft/fluentui-react-build
+/sizeauditor.json @microsoft/fluentui-react-build
 
 #### Repo stuff
-LICENSE @justSlone
-.github @justSlone
+/LICENSE @justSlone @jurokapsiar
```

Comment:
I don't understand this change either--is the license file at the root even used for anything? Wouldn't the licenses in the actual packages be the ones applicable to our actual published code?

Reply:
That's a good point that this license file doesn't have a ton of impact. Yes the LICENSE file in actual packages are the ones that are applicable. 

It is the license file that gets picked up by github to label the license for the repo, but that probably doesn't have any real impact. 

In the end. I don't see a strong reason to add extra scrutiny here beyond the regular code owners. Especially while we're adding a lot of packages, it's not worth the cost of adding an extra review. 

Let's keep this rule for now, since we don't have a codeowner for all files in the root